src/Examples/Flowsheets/ngcc/ngcc.ipynb

```diff
@@ -0,0 +1,2483 @@
+      "\n",
+      "iter    objective    inf_pr   inf_du lg(mu)  ||d||  lg(rg) alpha_du alpha_pr  ls\n",
+      "   0  0.0000000e+00 3.50e+01 1.00e+00  -1.0 0.00e+00    -  0.00e+00 0.00e+00   0\n",
+      "   1  0.0000000e+00 3.49e-01 8.68e+03  -1.0 2.71e+03    -  9.90e-01 9.90e-01h  1\n",
+      "   2  0.0000000e+00 3.20e-03 3.48e+02  -1.0 2.26e+03    -  9.89e-01 9.91e-01h  1\n",
+      "   3  0.0000000e+00 1.16e-04 1.03e+03  -1.0 4.07e+02    -  9.90e-01 1.00e+00h  1\n",
+      "   4  0.0000000e+00 1.03e-05 8.96e+03  -1.0 3.55e+01    -  9.90e-01 1.00e+00h  1\n",
+      "   5  0.0000000e+00 9.10e-07 5.99e+00  -1.0 3.14e+00    -  9.99e-01 1.00e+00h  1\n",
+      "\n",
+      "Number of Iterations....: 5\n",
+      "\n",
+      "                                   (scaled)                 (unscaled)\n",
+      "Objective...............:   0.0000000000000000e+00    0.0000000000000000e+00\n",
+      "Dual infeasibility......:   0.0000000000000000e+00    0.0000000000000000e+00\n",
+      "Constraint violation....:   9.0953117609648970e-07    9.0953117609648970e-07\n",
+      "Complementarity.........:   0.0000000000000000e+00    0.0000000000000000e+00\n",
+      "Overall NLP error.......:   9.0953117609648970e-07    9.0953117609648970e-07\n",
+      "\n",
+      "\n",
+      "Number of objective function evaluations             = 6\n",
+      "Number of objective gradient evaluations             = 6\n",
+      "Number of equality constraint evaluations            = 6\n",
+      "Number of inequality constraint evaluations          = 0\n",
+      "Number of equality constraint Jacobian evaluations   = 6\n",
+      "Number of inequality constraint Jacobian evaluations = 0\n",
+      "Number of Lagrangian Hessian evaluations             = 5\n",
+      "Total CPU secs in IPOPT (w/o function evaluations)   =      0.261\n",
+      "Total CPU secs in NLP function evaluations           =      4.609\n",
+      "\n",
+      "EXIT: Optimal Solution Found.\n"
+     ]
+    }
+   ],
+   "source": [
+    "m = pyo.ConcreteModel()\n",
+    "m.fs = ngcc.NgccFlowsheet(default={\"dynamic\":False})\n",
+    "iscale.calculate_scaling_factors(m)\n",
+    "m.fs.initialize(\n",
+    "    load_from=\"ngcc_init.json.gz\",\n",
+    "    save_to=\"ngcc_init.json.gz\",\n",
+    ")\n",
+    "res = solver.solve(m, tee=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "07da5349-dfdb-4ba2-adbe-e1e585405d95",
+   "metadata": {},
+   "source": [
+    "## Show PFDs with baseline results\n",
+    "\n",
+    "This displays PFDs in the notebook, and saves them to files.  The full NGCC model is too big to show well in a single PFD, so they are broken into the three main sections, gas turbine, heat recovery steam generator (HRSG), and steam turbine."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "id": "7328df70-190f-4ff3-8992-e8c23fa646a6",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
```

Comment:
Minor comment, but it might help to have a little bit more separation between each of the images here so that the sub-flowsheets are more easily separated. At the moment, it is a little hard to see where one ends and the next begins.